### PR TITLE
Add `jQuery`

### LINKED ISSUE #13: jQuery alternatives
The exact alternatives could be:

- native JS
- some more robust frameworks, like Angular, React, Vue
- some lighter alternatives, like [htmx ](https://htmx.org/)or [Alpine.js](https://alpinejs.dev/)

### PR DESCRIPTION
**Pull request by zloirock**
Fixes #13.

### CODE REVIEW

File: docs/modules/jquery.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# jQuery
+
+jQuery is a great project that at one time greatly simplified the life of web developers. However, now almost all features from this project can be easily replaced with native JavaScript methods and it makes no sense to load this quite a big dependence.
```
Comment (43081j) [inline docs/modules/jquery.md:3]: i would reword the last bit as:

> so it makes little sense to load a dependency this large to provide them

seems like `dependence` should be `dependency` at least

Comment (zloirock) [reply docs/modules/jquery.md]: Ok. Updated.

**Review by 43081j — APPROVED** [review]
